tools/performance-benchmarking/src/md/0010-content.md

```diff
@@ -0,0 +1,916 @@
+`benchcomp` allows you to:
+
+* Run two or more sets of benchmark suites under different configurations;
+* declaratively define visualizations to be generated from the resulting data;
+* run the tool with sensible defaults, or hook into every stage of the pipeline when you need more flexibility.
+
+This documentation contains three sections.
+The [user walkthrough](#user-walkthrough) takes you through the process of setting up an entirely new benchmark run and dashboard.
+The [developer reference](#developer-reference) describes `benchcomp`'s architecture and the different data formats that it uses, enabling you to author a benchmark run and dashboard of your own.
```

Comment:
We've primarily been using the "dashboard" term to mean a page that tracks performance over time. It's used here to refer to the results of a single benchcomp run, which confused me a bit. Perhaps replace it with "Visual results page" or something similar?